app/src/main/java/com/vultisig/wallet/data/models/SwapQuote.kt

```diff
@@ -26,4 +26,12 @@ internal sealed class SwapQuote {
         val data: THORChainSwapQuote,
     ) : SwapQuote()
 
-}
\ No newline at end of file
+}
+
+
+internal val SwapQuote.decimalsForRecommendedMinAmount: Int
+    get() = when(this){
+        is SwapQuote.MayaChain -> 8
+        is SwapQuote.ThorChain -> 8
+        is SwapQuote.OneInch -> 0
+    }
```

Comment:
@yvebe 

1inch does not have a minimum swap amount and I used 0 for it to make sure it pass check condition.

Token decimals should be ignored since both Thorchain and Mayachain use a fixed precision of 1e8, regardless of the token's native decimal places.

Reply:
in regards to decimals , both THORChain and MayaChain is 1e8 ,
The property name is `decimalsForRecommendedMinAmount` , using 0 is quite weird to be honest.

Reply:
Actually, 1inch does not have a minimum swap amount.
 
Therefore, the default value is 0, and it does not matter what value is used for the 1inch decimal. 
How about using 1 for the decimal?